v03_pipeline/lib/annotations/expression_helpers.py

```diff
@@ -0,0 +1,327 @@
+import hail as hl
```

Comment:
This file is entirely copy-pasta.  I kept the existing functions as is to avoid creating a dependency on `v03_pipeline` within the v2 pipeline.